docs/0_overview.md

```diff
@@ -0,0 +1,44 @@
+# Overview
+
+Crossplane is an open source control plane that allows you to manage
+infrastructure the Kubernetes way. It provides the following features:
+
+- Deployment and management of cloud provider managed services using the
+  Kubernetes API.
+- Management and scheduling of configuration data across multiple Kubernetes
+  clusters.
+- Separation of concern between [infrastructure owners](), [application
```

Comment:
It seems like all links are empty. Are you planning on filling them at a later point?

Reply:
Yep! See checklist in PR body :+1: